README.md

```diff
@@ -39,15 +39,12 @@ RMM can be installed with Conda ([miniconda](https://conda.io/miniconda.html), o
 [Anaconda distribution](https://www.anaconda.com/download)) from the `rapidsai` channel:
 
 ```bash
+# for CUDA 11.5
```

Comment:
@shwina I updated this -- I'm going to go ahead and make similar changes for other repos if you don't mind.